client/src/components/NavMenu/NavMenu.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import './NavMenu.css'
+
+const NavMenu = (props) => (
+    props.isVisible ?
+        <div className="nav-container">
+            <nav>
+                <ul>
+                    <li>
+                        <Link to="#">
+                            <h3 className="nav-title">Home</h3>
```

Comment:
Please fix your indentations here. Might be helpful for you to install a React linter.